src/lib/grammar.js

```diff
@@ -129,12 +129,13 @@ function buildRegex(map, keyEx, valEx) {
         '(?<' + keyEx + '>',
             keys,
         ')',
-        // value is optional
         '(?:\\(',
             '(?<' + valEx + '>',
                 GRAMMAR.VALUES,
             ')',
-        '\\))?',
+        '\\))',
+        // value is optional only for helper
+        keyEx === 'helper' ? '?' : ''
```

Comment:
Actually, this is an issue today. LineClamp is required to have params but we are making it optional because it's a helper. So invalid LineClamp constructs will match and we don't want that. I'll try to get it fixed.